src/main/webapp/assets/js/feedScript.js

```diff
@@ -785,21 +838,44 @@ function resetMarks(formElements) {
 function markInvalidInputs(invalidIds, errorMessages, formElements) {
   resetMarks(formElements);
   invalidIds.forEach((id) => {
-    const elt = formElements.namedItem(id);
```

Comment:
is `elt` element? maybe consider using `elmt` instead as it's not immediately apparent :)

Reply:
+1. elt isn't clear, prefer element.

Reply:
For readability, imagine someone totally new to your DB reading this, it wouldn't be as clear.

Reply:
Good point

Reply:
Good point, changed!